### PR TITLE
[Cassandra] Fix: cassandra.cluster.Error wasn't imported

### LINKED ISSUE #1422: NameError: global name 'cassandra' is not defined
Hey,

### Issue Summary
I added `export REDASH_ADDITIONAL_QUERY_RUNNERS=redash.query_runner.cass` to `opt/redash/.env`. After that, I was able to create a new cassandra datasource but when querying is failing and displaying: **global name 'cassandra' is not defined** 

I have installed the DataStax Cassandra driver `sudo pip install cassandra-driver` and using Python 2.7.6. 

These are the traces of `/opt/redash/logs/celery_error.log`

```
2016-11-23 02:09:15,625: WARNING/Worker-5] Unexpected error while running query:
Traceback (most recent call last):
  File "/opt/redash/redash.0.12.0.b2449/redash/tasks/queries.py", line 410, in run
    data, error = query_runner.run_query(annotated_query, self.user)
  File "/opt/redash/redash.0.12.0.b2449/redash/query_runner/cass.py", line 90, in run_query
    except cassandra.cluster.Error, e:
NameError: global name 'cassandra' is not defined
```
### Technical details

* Redash version 0.12.0+b2449
* OS Ubuntu 3.13.0-88-generic / Browser: Chrome.
* How did you install Redash: running the provisioning script

Thanks

### PR DESCRIPTION
**Pull request by arikfr**
Fixes #1422.

### CODE REVIEW

File: redash/query_runner/cass.py [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 
-from redash.query_runner import *
+from redash.query_runner import BaseQueryRunner, register
 from redash.utils import JSONEncoder
 
 logger = logging.getLogger(__name__)
 
 try:
-    from cassandra.cluster import Cluster
+    from cassandra.cluster import Cluster, Error
```
Comment (chemipot) [inline redash/query_runner/cass.py]: not fully sure if this `Error` import works well. At least when I tested, it did not work and I have to remove it and replace the reference below by `ValueError`.

File: redash/query_runner/cass.py [hunk 2]
```diff
@@ -86,22 +85,22 @@ def run_query(self, query, user):
             json_data = json.dumps(data, cls=JSONEncoder)
 
             error = None
-
-        except cassandra.cluster.Error, e:
+        except Error as e:
```
Comment (chemipot) [inline redash/query_runner/cass.py]: as commented above I change it for `ValueError`

File: redash/query_runner/cass.py (hunk 1 above)
Comment (arikfr) [inline redash/query_runner/cass.py]: What do you mean by works well? It wasn't found?

Comment (arikfr) [inline redash/query_runner/cass.py]: You're right there is no such thing. I just assumed it exists as it was referenced in the code... 

@yershalom ?

**Comment by yershalom** [comment]
Hi @arikfr 
Weird mistake, I pushed the fix.

**Pull request merged**